### PR TITLE
Add release notes for Chocolatey Central Management 0.9.0

### PR DESCRIPTION
**Pull request by gep13**
## Description Of Changes

Add release notes for Chocolatey Central Management 0.9.0

## Motivation and Context

There is a release coming up, and we need to get this documented

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

N/A

### CODE REVIEW

File: input/en-us/central-management/release-notes.md [hunk 1]
```diff
@@ -25,6 +25,42 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
 * Please see https://github.com/chocolatey/chocolatey-licensed-issues/labels/CentralManagement
 * Some issues may be held internally, please follow your support routes to learn more.
 
+## 0.9.0 (June 15, 2022)
+
+> :warning: **WARNING**
+>
+> The dependencies of all the Chocolatey Central Management packages (chocolatey-management-database, chocolatey-management-service, and chocolatey-management-web) have changed in this release. This is to allow the installation of .NET 6.0, which is now a requirement to run Chocolatey Central Management.
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:32]: Can you surround the package names with backticks?

Comment (pauby) [reply input/en-us/central-management/release-notes.md:32]: Can you also call out the remove of support for SQL Server 2008 / 2008 R2 here? I know it's also in Breaking Changes.

File: input/en-us/central-management/release-notes.md [hunk 2]
```diff
@@ -25,6 +25,42 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+
+### Breaking Changes
+
+- Add support for .NET 6.0 to all Chocolatey Central Management components
+  - As a result, the dependencies for all of the Chocolatey Central Management packages have changed
+- Update all Chocolatey Central Management packages to make use of commercial cmdlets and license validation
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:38]: Can you also call this out in the Warning section above too?

File: input/en-us/central-management/release-notes.md [hunk 3]
```diff
@@ -25,6 +25,42 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+  - This will mean that installation of the Chocolatey Central Management packages has to happen on a correctly licensed machine. [Further information about why we have enabled this can be found on our blog post](https://blog.chocolatey.org/2021/09/chocolatey-licensed-changes-restricted-to-licensed-nodes/)
+- Remove support for SQL Server 2008
+  - [Further information about this change can be found in our blog post](https://blog.chocolatey.org/2022/06/ccm-090-remove-sqlserver2008-support/)
+
+### Features
+
+- Add ability to update the "Friendly Name" of a computer via the API - see [Licensed #285](https://github.com/chocolatey/chocolatey-licensed-issues/issues/285)
+- Provide ability to ensure that all Chocolatey Central Management Chocolatey packages are not being installed to a SQL Server 2008 instance
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:46]: Can you add that includes 2008 R2 as well?

File: input/en-us/central-management/release-notes.md [hunk 4]
```diff
@@ -25,6 +25,42 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+
+### Bug Fixes
+
+- Ensure that it is possible to hide links from login page when new user registration is disabled - see [Licensed #270](https://github.com/chocolatey/chocolatey-licensed-issues/issues/270)
+- Ensure that no error is caused when viewing a large response from the website
+- Ensure that the chocolatey-management-web Chocolatey package can be internalized using Package Internalizer
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:52]: Can you add backticks around the package name?

File: input/en-us/central-management/release-notes.md [hunk 5]
```diff
@@ -25,6 +25,42 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+
+### Improvements
+
+- [Security] Remove usage of morris.js due to security vulnerability
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:56]: Can you put backticks around morris.js?

File: input/en-us/central-management/release-notes.md [hunk 6]
```diff
@@ -25,6 +25,42 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+- Add ability for the CCM Admin Role, by default, to be able to edit a computer - see [Licensed #217](https://github.com/chocolatey/chocolatey-licensed-issues/issues/217)
+- Add ability to see the "Friendly Name" assigned to a computer across the other parts of website - see [Licensed #221](https://github.com/chocolatey/chocolatey-licensed-issues/issues/221)
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:58]: Can you replace 'assigned to a computer across the other parts of website' with 'assigned to a computer across other areas of Chocolatey Central Management'?

File: input/en-us/central-management/release-notes.md [hunk 7]
```diff
@@ -25,6 +25,42 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+- Add colorization to the Deployment Plan / Step screens to show overall result in detail pages
+- Remove "Show Log" button from Deployment Step details page for a computer with a result reason of "Unreachable"
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:60]: No issues for these two?

**Review by pauby — CHANGES_REQUESTED** [review]

File: input/en-us/central-management/release-notes.md (hunk 7 above)
Comment (gep13) [reply input/en-us/central-management/release-notes.md:60]: No, these came up internally, and never as a support ticket, or at least, not that I could see from the issues.

**Review by pauby — APPROVED** [review]

**Pull request merged**